docs/user-guide/annotations.md

```diff
@@ -42,103 +38,135 @@ After selecting a geometry type, the editor can:
+
+* Enable the [**Compare tool**](toc.md#toolbar-options) through the <img src="../img/button/compare_tool_button.jpg" class="ms-docbutton" /> button
+
+* **Edit** the annotation through the <img src="../img/button/edit_button.jpg" class="ms-docbutton" /> button
+
+* **Download** the annotation through the <img src="../img/button/download_annotation_button.jpg" class="ms-docbutton" /> button
 
 ## Styling Annotations
 
-Based on which type of annotation was chosen, [MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) allows you to customize the annotation style through a powerful editor. It is accessible from the *Style* tab of the annotation viewer. During the style editing a preview placed on top of the styler form shows a preview of the edited style.
+Based on which type of annotation was chosen, [MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) allows you to customize the annotation style through a powerful editor. It is accessible from the *Style* tab of the annotation viewer.
 
 <img src="../img/annotations/annotations_toolbar.jpg" class="ms-docimage" style="max-width:500px;"/>
 
 ***Marker***
 
-MapStore provides two types of *Marker* annotations, so you have to choose what type do you prefer using the *Type* combo box (*Marker* is the default):
+MapStore provides different *Marker* annotation style:
+
+* The *Mark* types: clicking on the <img src="../img/button/add_mark_button.jpg" class="ms-docbutton"/> button, a mark panel appears.
 
 <img src="../img/annotations/marker_type_selection.jpg" class="ms-docimage" style="max-width:500px;"/>
 
-* *Marker* types can be customized through the following editor:
+The mark can have different `Shape`, `Color`, `Stroke` with different `Color`, `Width` and `Style` and customizable `Radius` and `Rotation`. Take a look at the following example.
 
 <img src="../img/annotations/marker_style_editor.jpg" class="ms-docimage" style="max-width:500px;"/>
 
-Choose the `Shape`, `Color` and `Icon` that best fit your needs.
+* The *Icon* types: clicking on the <img src="../img/button/add_icon_button.jpg" class="ms-docbutton"/> button, an icon panel appears.
+The icon can have different `Image`. Clicking on it, the user can choose a *Marker* and a *Glyph*, as follows:
 
-* *Symbol* types can  have different `Shape` and `Size`, a `Fill color` with a customizable `Opacity` level (%), a `Stroke` of different types (continuous, dashed, etc) and customizable `Color`, `Opacity` and `Width`. Only few symbols are provided by default in MapStore but a custom list of symbols can be configured.
+<video class="ms-docimage" controls><source src="../img/annotations/icon_style_editor.mp4" ></video>
 
-<img src="../img/annotations/symbol_style_editor.jpg" class="ms-docimage" style="max-width:500px;"/>
+The icon is also customizable with different `Opacity`, `Size`, `Rotation` and `Anchor point`.
 
```

Comment:
@ElenaGallo please put a note here indicating the other symbolizer for the 3D with a link to the related section.